src/sqlfluff/core/linter/linter.py

```diff
@@ -519,6 +519,14 @@ def render_string(
         # Start the templating timer
         t0 = time.monotonic()
 
+        if not config.get("templater_obj") == self.templater:
+            linter_logger.warn(

```

Comment:
codecov is complaining here, this is a very useful message so I'd like to keep it ^^

Could you add a test that just checks the templater not being updated by .sqlfluff in a subdir?

Reply:
@dmateusp I've added a test to cover these lines [here](https://github.com/sqlfluff/sqlfluff/pull/898/files#diff-fe70b9c424984a64d62d1fd3dc9c3b3877ea0d2de3331b2f27fd56863bd57520R595)